packages/suite/src/components/wallet/InputInfo.tsx

```diff
@@ -0,0 +1,45 @@
+import { MouseEventHandler } from 'react';
+import styled from 'styled-components';
+import { useTheme } from 'styled-components';
+import { Button } from '@trezor/components';
+import { LearnMoreButton } from '../suite/LearnMoreButton';
+import { TooltipSymbol } from '../suite';
+import { getInputStateTextColor } from '@trezor/components';
+const Wrapper = styled.div`
+    display: flex;
+    align-items: center;
+    gap: 8px;
+    width: 100%;
+    justify-content: space-between;
+`;
+
+interface InputInfoProps {
+    button?: { onClick: MouseEventHandler<HTMLButtonElement>; text: string } | { url: string };
```

Comment:
I know that you put this here for future usage and because you want to approach it in generic way but since you don't use this at all I think you can remove it completely

Reply:
oki, yep I thought it might be useful in the future, but we can add it later in case we need it. removed